src/models/book.ts

```diff
@@ -0,0 +1,26 @@
+import { LendingRecord } from "./lendingRecord";
+import { v4 as uuidv4 } from "uuid";
```

Comment:
instead of importing this lib every place we need it, let's create an utils.ts file and create a method for creating IDs using that lib there. Then we can simply import the method and use it where needed

Reply:
Nice approach, I will implement it!